### ISSUE TITLE
Questions about letterbox and mosaic

### ISSUE THREAD

**Issue by starspder** [AlexeyAB/darknet#8526]
I used yolov4 for training and had some questions while training so far. I know that letter_box=1 and mosaic=1 generally perform better than when not applied. I wonder if better performance was experimentally obtained than when 0 was given by setting letter_box=1, mosaic=1 even during darknet training.

I don't know if my computer is weird, but the learning speed is noticeably slow when I give it mosaic=1 and letter_box=1. So if there's no difference in performance, I'd like to give it mosaic=0 and letter_box=0., but I wonder what you think about this.

**Comment by stephanecharette** [comment]
I don't believe letterbox has been supported in several years, and mosaic degrades network accuracy (see issue #6105) so I typically leave it at zero.

**Comment by agjunyent** [comment]
Mosaic improves generalization for large models, that allow "large" input network sizes. If you have a size of 640x640 or more, I always find mosaic to improve generalization, with an increase in loss and training time and slight decrease in mAP
